test/common_methods_invocations.py

```diff
@@ -797,6 +797,14 @@ def method_tests():
     ]
 # TODO: clamp with min/max
 
+def torch_function_tests():
+    return [
+        ('squareform', (1,), NO_ARGS, '1d'),
```

Comment:
the point of this file is to automatically run these tests through different functionality, i.e. autograd, jit, etc.  This won't be run through the JIT tests -- does the JIT work if you add these tests to the above list?